src/SDL2.cs

```diff
@@ -29,6 +29,7 @@
 #region Using Statements
 using System;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
I think this'l need to be guarded as well?

Reply:
Added